src/http/ngx_http_wasm_api.c

```diff
@@ -679,18 +765,21 @@ ngx_http_wasm_req_get_headers(ngx_http_request_t *r, int32_t addr, int32_t size_
         ngx_http_wasm_h2_header_t   *wh;
```

Comment:
bad style.

We should put it at the beginning of the function.

Reply:
updated